package.json

```
@@ -37,14 +37,15 @@
   },
   "homepage": "https://medibloc.org",
   "dependencies": {
-    "@cosmjs/amino": "0.25.5",
-    "@cosmjs/crypto": "0.25.5",
-    "@cosmjs/encoding": "0.25.5",
-    "@cosmjs/math": "0.25.5",
-    "@cosmjs/proto-signing": "0.25.5",
-    "@cosmjs/stargate": "0.25.5",
-    "@cosmjs/tendermint-rpc": "0.25.5",
-    "@cosmjs/utils": "0.25.5",
+    "@cosmjs/amino": "^0.27.1",
+    "@cosmjs/crypto": "^0.27.1",
+    "@cosmjs/encoding": "^0.27.1",
+    "@cosmjs/launchpad": "^0.27.1",
+    "@cosmjs/math": "^0.27.1",
+    "@cosmjs/proto-signing": "^0.27.1",
+    "@cosmjs/stargate": "^0.27.1",
+    "@cosmjs/tendermint-rpc": "^0.27.1",
+    "@cosmjs/utils": "^0.27.1",
```

```suggestion
    "@cosmjs/amino": "0.27.1",
    "@cosmjs/crypto": "0.27.1",
    "@cosmjs/encoding": "0.27.1",
    "@cosmjs/launchpad": "0.27.1",
    "@cosmjs/math": "0.27.1",
    "@cosmjs/proto-signing": "0.27.1",
    "@cosmjs/stargate": "0.27.1",
    "@cosmjs/tendermint-rpc": "0.27.1",
    "@cosmjs/utils": "0.27.1",
```
Please pin dependencies. As I know, cosmjs doesn't guarantee backward-compatibility now.